components/fonts/platform/freetype/font.rs

```diff
@@ -148,11 +155,15 @@ impl PlatformFontMethods for PlatformFont {
             return Err("Could not create FreeType face");
         }
 
+        // Set font size
```

Comment:
Nit: I think you can probably omit this comment as we are calling a method called `set_size` on `font` below, which is a bit self explanatory.